### PR TITLE
Added css, readme, and html files

### PR DESCRIPTION
**Pull request by c00p75**
Added css, readme, and html files

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+body{
```
Comment (c00p75) [inline style.css:3]: Adding a space between `{` and the html element is good practice.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  margin: 0;
+  /*background:linear-gradient(90deg, rgb(22, 22, 78), rgb(207, 207, 223), rgb(255, 255, 255), rgb(207, 207, 223), rgb(22, 22, 78));*/
+  background-color: whitesmoke;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  font-family: 'Lato', sans-serif; 
+
+
```
Comment (c00p75) [inline style.css:11]: I suggest removing unused white spaces in the code. The code will be standardized and look cleaner

File: style.css (hunk 1 above)
Comment (c00p75) [reply style.css:3]: Adding a space between { and the html element is good practice.

I suggest removing unused white spaces in the code. The code will be standardized and look cleaner

Comment (c00p75) [inline style.css:3]: I suggest adding a space between `{` and all html elements in css. It is a linter requirement and is good practice.

File: style.css (hunk 2 above)
Comment (c00p75) [inline style.css:11]: It is better to remove unused lines and white spaces from the code. The code will be standardized and look cleaner

File: style.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  min-width:50vw;
+}
+
+#div1{  
+  margin-top: -10px;
+  padding: 0% 34vw;
+  background-image: url(https://wallpaperaccess.com/full/4199381.jpg);  
+  opacity: .7;
+}
+
```
Comment (c00p75) [inline style.css:27]: It is better to remove unused lines and white spaces from the code. The code will be standardized and look cleaner

File: style.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+header{ 
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  color: rgb(255, 255, 255);
+  margin: auto;
+  opacity: 1;
+  text-align: center;
+  border-radius: 7px;
+  margin-top: -0px;
+
```
Comment (c00p75) [inline style.css:48]: It is better to remove unused lines and white spaces from the code. The code will be standardized and look cleaner

File: style.css [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  font-size: 0.5cm;
+  opacity: 1;
+  background:rgba(245, 245, 245, 0.9);   
+  border-radius: 10px;
+  margin: 5% -30%;
+  padding: 5% 15% 5% 15%;
+  line-height:0.8cm
+}
+
+
```
Comment (c00p75) [inline style.css:62]: It is better to remove unused lines and white spaces from the code. The code will be standardized and look cleaner

File: style.css [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+h1{font-weight: bold;
+   font-size: 50px;
```
Comment (c00p75) [inline style.css:64]: I recommend starting your code after `{` indented on the next line. This will improve readability.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+}
+
```
Comment (c00p75) [inline style.css:66]: It is better to remove unused lines and white spaces from the code. The code will be standardized and look cleaner

File: style.css [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+  font-weight: bold;
+  font-size: 0.5cm;
+  margin: auto;
+  width:100%;
+  padding: 0.75rem 10%;
+  border-radius: 5px;
+  border: 2px solid rgb(80, 89, 139);
+  background: rgb(80, 89, 139);
+  cursor: pointer;
+}
```
Comment (c00p75) [inline style.css:124]: I further suggest adding a line after your last `}` in order to pass the linter requirement

File: surveypage.html [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <title>Survey Form</title>
+      <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+      <div id='div1'>
+        <header>
+          <h1 id = "title">CUSTOMER FEEDBACK</h1>
```
Comment (c00p75) [inline surveypage.html:14]: I advise using space-less equal signs. This will make your code look concise.

File: surveypage.html [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+            <p id="description">We would love to hear your thoughts, suggestions, or concerns regarding your shopping experience.</p>
+        </header>
+      </div>
+      <div id="div2">
+        <main>
+          <form id="survey-form" method = "get">
```
Comment (c00p75) [inline surveypage.html:20]: I advise using space-less equal signs. This will make your code look concise.

**Review by c00p75 — COMMENTED** [review]
# Changes Required ♻️ 

Good day. 

Please read the comments below for the required changes. 

Be sure to re-submit the project once the changes have been made.

Thank you 🙌

**Review by sajid-munawar — APPROVED** [review]
### Hi @c00p75 ,
Great job on this project review! 👍🏻
# Status: Approved ✔️
Excellent job on this project review. Keep it up 👏
Your project review is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
>
### To Highlight
- [x]  Added descriptive inline comments. :heavy_check_mark: 
- [x]  Review comments are polite, use suggestions and give reasons for requested change :heavy_check_mark:
- [x] Made use of emojis in project review summary to make the code review more human-friendly 👽🤖. :heavy_check_mark: 
- [x] Made use of [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) :heavy_check_mark: 

 ## [Optional] suggestions:
 * Nothing to mention.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**